src/WebJobs.Script/functionTemplate.js

```diff
@@ -4,14 +4,27 @@
 var f = require('{0}');
 
 return function (context, callback) {{
-    context.done = callback;
+    context.done = function(err, result) {{
```

Comment:
This complexity in the scaffolding comes from the fact that Edge.js doesn't marshal any properties set on the context back to .NET for us automatically. So here, we're aggregating them ourselves into a 'values' object and passing them back to ourselves via a context.bind helper function.